.github/workflows/ci-run-on-pr.yaml

```
@@ -148,18 +148,99 @@ jobs:
         shell: bash
         run: echo "coverage check failed" && exit 1
 
+  e2e-keeper-images:
+    name: Build keeper images for running e2e
+    runs-on: ubuntu-22.04
+    needs:
+      - codechanges
+    if: ${{ (needs.codechanges.outputs.backend == 'true' || needs.codechanges.outputs.ci == 'true') && github.event.pull_request.draft == false}}
```

actionlint (1.7.10)

[error] property "ci" is not defined in object type {backend: string}

(expression)